From 7cd7a43eff6ff593b268140cfe7e6ae96a4e8cab Mon Sep 17 00:00:00 2001
From: Shizhao Liu <lshizhao@amazon.com>
Date: Fri, 5 Sep 2025 11:51:42 -0700
Subject: [PATCH 15/15] Fix iso url for ubunutu 24.04

The iso_url for Ubuntu 24.04 used by upstream image-builder is unavailable,
changing it to the correct iso url.
---
 images/capi/packer/ova/ubuntu-2404-efi.json     | 4 ++--
 images/capi/packer/ova/ubuntu-2404.json         | 4 ++--
 images/capi/packer/raw/raw-ubuntu-2404-efi.json | 4 ++--
 images/capi/packer/raw/raw-ubuntu-2404.json     | 4 ++--
 4 files changed, 8 insertions(+), 8 deletions(-)

diff --git a/images/capi/packer/ova/ubuntu-2404-efi.json b/images/capi/packer/ova/ubuntu-2404-efi.json
index ea0930bf9..fa88477ea 100644
--- a/images/capi/packer/ova/ubuntu-2404-efi.json
+++ b/images/capi/packer/ova/ubuntu-2404-efi.json
@@ -9,9 +9,9 @@
   "firmware": "efi",
   "floppy_dirs": "./packer/ova/linux/{{user `distro_name`}}/http/",
   "guest_os_type": "ubuntu-64",
-  "iso_checksum": "d6dab0c3a657988501b4bd76f1297c053df710e06e0c3aece60dead24f270b4d",
+  "iso_checksum": "c3514bf0056180d09376462a7a1b4f213c1d6e8ea67fae5c25099c6fd3d8274b",
   "iso_checksum_type": "sha256",
-  "iso_url": "https://releases.ubuntu.com/noble/ubuntu-24.04.2-live-server-amd64.iso",
+  "iso_url": "https://releases.ubuntu.com/24.04.3/ubuntu-24.04.3-live-server-amd64.iso",
   "os_display_name": "Ubuntu 24.04",
   "shutdown_command": "shutdown -P now",
   "vsphere_guest_os_type": "ubuntu64Guest"
diff --git a/images/capi/packer/ova/ubuntu-2404.json b/images/capi/packer/ova/ubuntu-2404.json
index 0f910b6eb..4fbbb6061 100644
--- a/images/capi/packer/ova/ubuntu-2404.json
+++ b/images/capi/packer/ova/ubuntu-2404.json
@@ -8,9 +8,9 @@
   "distro_version": "24.04",
   "floppy_dirs": "./packer/ova/linux/{{user `distro_name`}}/http/",
   "guest_os_type": "ubuntu-64",
-  "iso_checksum": "d6dab0c3a657988501b4bd76f1297c053df710e06e0c3aece60dead24f270b4d",
+  "iso_checksum": "c3514bf0056180d09376462a7a1b4f213c1d6e8ea67fae5c25099c6fd3d8274b",
   "iso_checksum_type": "sha256",
-  "iso_url": "https://releases.ubuntu.com/noble/ubuntu-24.04.2-live-server-amd64.iso",
+  "iso_url": "https://releases.ubuntu.com/24.04.3/ubuntu-24.04.3-live-server-amd64.iso",
   "os_display_name": "Ubuntu 24.04",
   "shutdown_command": "shutdown -P now",
   "vsphere_guest_os_type": "ubuntu64Guest"
diff --git a/images/capi/packer/raw/raw-ubuntu-2404-efi.json b/images/capi/packer/raw/raw-ubuntu-2404-efi.json
index c9d93a0bc..b17f0356f 100644
--- a/images/capi/packer/raw/raw-ubuntu-2404-efi.json
+++ b/images/capi/packer/raw/raw-ubuntu-2404-efi.json
@@ -8,9 +8,9 @@
   "distro_version_slug": "2404",
   "firmware": "OVMF.fd",
   "guest_os_type": "ubuntu-64",
-  "iso_checksum": "d6dab0c3a657988501b4bd76f1297c053df710e06e0c3aece60dead24f270b4d",
+  "iso_checksum": "c3514bf0056180d09376462a7a1b4f213c1d6e8ea67fae5c25099c6fd3d8274b",
   "iso_checksum_type": "sha256",
-  "iso_url": "https://releases.ubuntu.com/releases/24.04/ubuntu-24.04.2-live-server-amd64.iso",
+  "iso_url": "https://releases.ubuntu.com/24.04.3/ubuntu-24.04.3-live-server-amd64.iso",
   "os_display_name": "Ubuntu 24.04",
   "shutdown_command": "shutdown -P now",
   "unmount_iso": "true"
diff --git a/images/capi/packer/raw/raw-ubuntu-2404.json b/images/capi/packer/raw/raw-ubuntu-2404.json
index 607b25342..9f2afe142 100644
--- a/images/capi/packer/raw/raw-ubuntu-2404.json
+++ b/images/capi/packer/raw/raw-ubuntu-2404.json
@@ -7,9 +7,9 @@
   "distro_version": "24.04",
   "distro_version_slug": "2404",
   "guest_os_type": "ubuntu-64",
-  "iso_checksum": "d6dab0c3a657988501b4bd76f1297c053df710e06e0c3aece60dead24f270b4d",
+  "iso_checksum": "c3514bf0056180d09376462a7a1b4f213c1d6e8ea67fae5c25099c6fd3d8274b",
   "iso_checksum_type": "sha256",
-  "iso_url": "https://releases.ubuntu.com/releases/24.04/ubuntu-24.04.2-live-server-amd64.iso",
+  "iso_url": "https://releases.ubuntu.com/24.04.3/ubuntu-24.04.3-live-server-amd64.iso",
   "os_display_name": "Ubuntu 24.04",
   "shutdown_command": "shutdown -P now",
   "unmount_iso": "true"
-- 
2.49.0

